influxdb3_client/src/lib.rs

```diff
@@ -128,6 +128,37 @@ impl Client {
     ) -> QueryRequestBuilder<'_> {
         QueryRequestBuilder {
             client: self,
+            kind: QueryKind::Sql,
+            db: db.into(),
+            query: query.into(),
+            format: None,
+        }
+    }
+
+    /// Compose a request to the `/api/v3/query_sql` API
```

Comment:
👋 Is this meant to be `api/v3/query_influxql`?

Reply:
It sure is! Thanks for the catch, I will fix this up.